sound/soc/sof/compress.c

```diff
@@ -36,7 +59,219 @@ void snd_sof_compr_fragment_elapsed(struct snd_compr_stream *cstream)
 		return;
 	}
 
+	sof_set_transferred_bytes(sstream, spcm->stream[cstream->direction].posn.host_posn,
+				  runtime->buffer_size);
```

Comment:
I have a doubt here on what the total bytes means, I don't recall this part.
a) is this the number of bytes provided to the device (application pointer)
b) or is this the number of bytes handled by the DMA (hw_ptr).

you seem to have taken option b), need to make sure it's the correct one :-)

Reply:
@plbossart do you mean Host dma? I couldn't figure out really well what posn.host_posn means in SOF FW Host component. I've asked this question here: https://github.com/thesofproject/sof/issues/4064

sofset_transferred_bytes can be further extended so that each platform have its own implementation.